tasks/config.yml

```diff
@@ -52,13 +52,30 @@
     path: "{{ nifi_emqtt_cert_dir }}"
   when: nifi_create_mqtt_certs
 
-- name: Create mqtt SSL Cerificates
-  java_keystore:
-    name: certificates
-    certificate: "{{ nifi_emqtt_certificate_authority_cert }}"
-    private_key: "{{ nifi_emqtt_certificate_authority_key }}"
-    password: "{{ nifi_emqtt_secure_password }}"
-    dest: "{{ nifi_emqtt_kestore_file_dest }}"
+- name: Copy emqtt certificate and key
+  copy:
+    content: "{{ nifi_emqtt_certificate_authority_key }}\n{{ nifi_emqtt_certificate_authority_cert }}"
+    dest: "{{ nifi_emqtt_cert_dir }}/key_cert.pem.txt"
```

Comment:
Do we need to keep this on the server after we are done?

Reply:
Probably not. Let me get rid of them.